backend/src/main/java/edu/uw/cs403/plantmap/backend/Application.java

```diff
@@ -38,6 +38,7 @@ public static void main(String[] args) {
         get("/v1/submission/:id",subCtr::getPost);
         delete("/v1/submission/:id", subCtr::deletePost);
         get("/v1/submission", subCtr::getAllPost);
+        post("/v1/submission/:id", subCtr::reportPost);
```

Comment:
We should change this so that it reports when we push to `/v1/submission/:id/report`.

Reply:
done